Development-Guide/AsciiDoc-Guide/README.adoc

```diff
@@ -0,0 +1,57 @@
+= AsciiDoc
+
+== Headers
+When writing an AsciiDoc a large header can be made by writing
+
+....
+= A Desciptive Title
+....
+
+Similarly a sub header can be created by writing
+
+....
+== A Descriptive Subtitle
+....
+
+== Lists
+
+An unordered list, ie. a list of bullet points can be made with
+[source,adoc]
+* First Item
+* Second Item
+
+A numbered list can be made as
+
+[source,adoc]
+. Step 1
+. Step 2
+
+Nested steps can be added by increasing the number of periods before the step.  For example:
+
+[source,adoc]
+. Step 1
+.. Step 1a
+. Step 2
+
+== Source Code
+
+To write source code in an AsciiDoc, write "[source,a language]"
+Lines after the square brackets will be displayed as a block of code.
+An empty line will end the block of code.
+For example:
+
+....
+[source, c]
+Some code
+Some more code
+....
+
+== Formatting
+
+In AsciiDocs it is best practice for each new sentence to be started on a new line in your text editor.
```

Comment:
AsciiDocs isn't really a term.

`When writing in asciidoc, each new sentence...`